pkg/deployment/deployment.go

```diff
@@ -33,6 +33,20 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
```

Comment:
No need to define them here.. There are there in the API for ansibleee and exported https://github.com/openstack-k8s-operators/openstack-ansibleee-operator/pull/171/files